cmd/create.go

```diff
@@ -44,6 +49,7 @@ func init() {
 	createCmd.Flags().StringP("remote-tcp", "c", "", `Remote host for inlets-pro to use for forwarding TCP connections`)
 
 	createCmd.Flags().DurationP("poll", "n", time.Second*2, "poll every N seconds, use a higher value if you encounter rate-limiting")
+	createCmd.Flags().BoolVarP(&staticFile, "static-file", "", false, "Create a static file for the provisioned machine")
```

Comment:
I'm having second thoughts on the naming.

`--static-file` is rather verbose and we should have a shortcut for it too like we have `-c` for `--remote-tcp`

What other ideas do you have? cc @Waterdrips

Reply:
@alexellis @Waterdrips Maybe --sf ?